### ISSUE TITLE
Sound toggle button

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#177]
Besides abstraction (mentioned in https://github.com/phetsims/john-travoltage/issues/149), it seems like:

1. The accessibleContent should be declared in the common-code SoundToggleButton
2. ```'Mute Sound'``` should be in the A11yStrings equivalent for whatever repo the accessible content lives in.

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jonathanolson** [comment]
Also the following code:
```js
    // when the sound property changes, toggle the 'aria-pressed' state
    // linked lazily as the parallel domElement won't exist in the document until
    // after instantiation
    model.soundProperty.lazyLink( function( pressed ) {
      document.getElementById( 'sound-button' ).setAttribute( 'aria-pressed', !pressed );
    } );
```
would be included.

It also seems suspicious to require finding by ID, we should have a direct reference.

Additionally, this would break if we toggled soundProperty after this but before the peer was created. If for some reason we can't get a direct reference (we should discuss), then the code should guard around the fact that the getElementById result could be null.

**Comment by jessegreenberg** [comment]
@jonathanolson I think these concerns should be fixed in the above commits.  Would you mind verifying?

SoundToggleButton implements the accessible content, Mute Sound has been added to SceneryPhetA11yStrings, getElementById() has been removed since we can set attributes directly now.

**Comment by jonathanolson** [comment]
Looks good, thanks.
